src/index.js

```diff
@@ -0,0 +1,151 @@
+const state = {
+    currentTemp: 70,
+    backgroundColor: document.getElementById('buttom--left'),
+    landscapeImg: document.getElementById('buttom--right'),
+    cityInput: document.getElementById('city-input'),
+    location: null,
+    lat: null,
+    lon: null,
+    skyCondition: document.getElementById('sky--options'),
+    skyImg: document.getElementById('top')
+}
+// changes the temp on the page
+const changesTempDisplay = () => {
+    const tempControl = document.getElementById('temperature');
+    tempControl.textContent = `${state.currentTemp}`
+}
+// CHANGES TEMP
+const increaseTemp = () => {
+    state.currentTemp++;
+    changesTempDisplay()
+};
+
+const decreaseTemp = () => {
+    state.currentTemp--;
+    changesTempDisplay()
+};
+
+// sky background change with sky option
+const skyBackground = () => {
+    console.log('skyBackground')
+    const condition = state.skyCondition.value
+    if(condition == 'sunny') {
+        console.log('sunny')
+        state.skyImg.style.backgroundImage = 'url(assets/sky/kyaw-zay-ya-8vHAfhWoqEQ-unsplash.jpg)';
+    } else if(condition == 'rainy') {
+        state.skyImg.style.backgroundImage = 'url(assets/sky/eutah-mizushima-F-t5EpfQNpk-unsplash.jpg)';
+    } else if(condition == 'snowy'){
+        state.skyImg.style.backgroundImage = 'url(assets/sky/fabrizio-conti-Mbm0WnJ5emc-unsplash.jpg)';
+    } else if(condition == 'cloudy') {
+        state.skyImg.style.backgroundImage = 'url(assets/sky/daoudi-aissa-Pe1Ol9oLc4o-unsplash.jpg)';
+    }
+
+    }
```

Comment:
Nitpick - indentation is off, line 43 should be unindented one level to the left